recipe/meta.yaml

```diff
@@ -53,7 +53,7 @@ outputs:
         - colorama >=0.4,<1.0
         - graphene >=2.1,<3
         - jinja2 >=3.0,<3.1
-        - metomi-isodatetime >=1!3.0.0, <1!3.1.0
+        - metomi-isodatetime >=1!3.0.0, <1!3.2.0
```

Comment:
I think this needs updating in conda-environment.yml too

Reply:
It was

Reply:
I think @MetRonnie  is correct.